week-1/1-exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var stuPercentile = numberOfStudents/23*100;
```

Comment:
For readability try to use full words unless it is a very well recognised acronym like `ID` for example
`studentPercentile`